src/toolbar/index.js

```diff
@@ -8,111 +8,115 @@ import { Analytics } from './analytics';
 import { PreviewCookie } from './preview/cookie';
 
 const version = process.env.npm_package_version;
+const IS_EMBEDDED = window.self !== window.top;
+
+if (!IS_EMBEDDED) {
```

Comment:
Alternatively, we could change this to a guard and prevent the extra indention and large diff:

```ts
if (IS_EMBEDDED) {
  return
}
```

Totally up to you since both approaches are valid. 🙂

Reply:
I'm not sure it's legal to return top-level in a script, could yield `SyntaxError: Illegal return statement`